### ISSUE TITLE
[Frontend] Vacation days overview

### ISSUE THREAD

**Issue by olemp** [Puzzlepart/did#265]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
Vacation days overview. Showing days used and days left of vacation e.g. on frontpage.

### Description
Show an overview of vacation days on e.g. frontpage. For version 1.0 it will just be a simple module/component showing vacation days used and remaining. It can be expanded to a separate page in the future if needed (but we don't need to overengineer it).

Quote @damsleth:

> love it

> det er faktisk dødshot feature

What we'll need:

- [ ] A `Vacation` / `VacationSummary` component
- [ ] A `graphql` endpoint `vacation` that returns e.g. `usedHours` and `remainingHours`
- [ ] Settings field on `Subscriptions` in azure table storage with number of vacation days and category for vacation events (`IAM VAC` for Puzzlepart)

#### graphql endpoint
We can't retrieve all events, so we have to do a `$filter` in the query to the graph.

Something along these lines:

```
https://graph.microsoft.com/v1.0/me/events?$filter=categories/any(a:a+eq+'IAM VAC') or startswith(subject,'IAM VAC')
```

So it's looking for `IAM VAC` in either the start of the `subject` or in `categories`.
